backend/audit/cross_validation/number_of_findings.py

```diff
@@ -0,0 +1,52 @@
+from .errors import (
+    err_number_of_findings_inconsistent,
+)
+from collections import defaultdict
+
+
+def number_of_findings(sac_dict):

```

Comment:
```suggestion
def number_of_findings(sac_dict, *_args, **_kwargs):

```
In another branch we realized that we might need a little more flexibility with what we pass to these functions, so all of them should actually have this signature; I think your existing tests and other code will still work fine with this change.